### PR TITLE
Finalizing glossary

### PR DESCRIPTION
**Pull request by chanan-sack**
changed different definitions, added others
it is not final but existing content is suitable.

### CODE REVIEW

File: glossary.asciidoc [hunk 1]
```diff
@@ -109,14 +109,10 @@ Frontier::
   The initial test development stage of Ethereum, lasted from July 30th 2015 to March 2016.
 
 Ganache::
-  A personal blockchain for Ethereum development. One can use Ganache to deploy contracts, develop applications, and run tests. It is available as a desktop application for Windows, Mac, and Linux.
+  personal Ethereum blockchain which you can use to run tests, execute commands, and inspect state while controlling how the chain operates.
```
Comment (fjrojasgarcia) [inline glossary.asciidoc]: Should not you capitalize the first letter?

**Review by fjrojasgarcia — CHANGES_REQUESTED** [review]

File: glossary.asciidoc (hunk 1 above)
Comment (chanan-sack) [reply glossary.asciidoc]: will be done and re-pr'd

File: glossary.asciidoc [hunk 2]
```diff
@@ -109,14 +109,10 @@ Frontier::
...
+  it is an extremely fast tool for executing transactions and automated testing of smart contracts.
```
Comment (aantonop) [inline glossary.asciidoc]: Capitalization and syntax. This sentence is hanging, without a capitalized first letter. Was it meant to follow a comma, or as a new sentence?

**Review by aantonop — CHANGES_REQUESTED** [review]
capitalization and syntax

File: glossary.asciidoc [hunk 3]
```diff
@@ -335,14 +331,12 @@ Transaction::
   Data committed to the Ethereum Blockchain signed by an originating account, targeting a specific address. The transaction contains metadata such as the gas limit for the transaction.
 
 Truffle::
-  The most popular Ethereum Development Framework. It is composed of several NodeJS packages and can be installed using Node Package Manager (NPM).
 
-////
+  The most popoular Ethereum Development Framework. It is composed of several NodeJS packages and can be installed using Node Package Manager (NPM).
```
Comment (fjrojasgarcia) [inline glossary.asciidoc:335]: Sorry, I don't know the meaning of **"popoular"**. :-=

**Review by fjrojasgarcia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**